.gitignore

```diff
@@ -1,11 +1,11 @@
 *.pyc
-_trial_temp
-build
+/_trial_temp/
```

Comment:
The goal later is to delete these, but I'm trying to make the changes incrementally.
I do't know what generates `htmlcov`, so I don't know if it's safe to pin to a location or remove…